internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -33,12 +33,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
+	apimacherrors "k8s.io/apimachinery/pkg/util/errors"
```

Comment:
Interesting: I thought our goimportas linter configuration would've had a package import enforcement for `utilerrors` but I guess I never added that. I'm fine with the import name as long as we're consistent throughout the codebase, and it doesn't look like we use this package anywhere else right now.